### DISCUSSION TITLE
[v0.23rc] Silence /api/health endpoint log entries

### DISCUSSION THREAD

**Discussion by pnmcosta** [pocketbase/pocketbase#5724] [Show and tell]
As per title, here's the snippet:

```
func(e *core.ServeEvent) error {
	// silence health endpoint
	if e.Router.HasRoute(http.MethodGet, "/api/health") {
		e.Router.BindFunc(func(e *core.RequestEvent) error {
			if e.Request.URL.Path == "/api/health" {
				return apis.SkipSuccessActivityLog().Func(e)
			}
			return e.Next()
		})
	}

	return e.Next()
},
```

To omit these:
![image](https://github.com/user-attachments/assets/bb63c92e-f83b-4054-b59b-d5989302a69f)

**Comment by ganigeorgiev** [comment]
~I'll consider disabling the success `/api/health` logs with the final release.~

**Reply by ganigeorgiev** [reply]
On second thought - I'll keep it as it is as the health action is public and it is better the log to be enabled by default so that you can inspect who is accessing it.
